### PR TITLE
Add test for steam GUI

### PR DESCRIPTION
**Pull request by okurz**
Verification run: http://lord.arch/tests/717

Related needle PR:
https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/345

### CODE REVIEW

**Comment by asmorodskyi** [comment]
you create to conflicting PRs , please solve this

**Comment by okurz** [comment]
They are not conflicting. If the reviewers are happy with all changes then merge either and the other PRs are either implicitly included or the other PRs only include the later commits but so far I do not see any conflicts.

**Comment by asmorodskyi** [comment]
ok , I was just in doubt how git will react here.

**Comment by DimStar77** [comment]
Steam is not available on non-intel archs (seen a failure on aarch64 already)

File: tests/installation/addon_products_via_SCC_yast2.pm [hunk 1]
```diff
@@ -35,14 +36,13 @@ sub test_setup {
         # Define proxy SCC
         assert_script_run "echo \"@addon_proxy.proxy.scc.suse.de\" > /etc/SUSEConnect";
     }
-    # Disable screensaver, fails sometimes on SLE12, so keep it as before without assertion
-    script_run "gsettings set org.gnome.desktop.session idle-delay 0";
+    turn_off_gnome_screensaver;
```
Comment (coolo) [inline tests/installation/addon_products_via_SCC_yast2.pm]: you're replacing script_run with type_string, i.e. no check if the command is finished before continueing

please revert

**Comment by coolo** [comment]
For a PR that changes so many modules, this is misses quite a lot of validation runs. Don't you think?

**Comment by okurz** [comment]
yes, you are right. I overlooked the impact of the screensaver moving commit. For the other commits I do not see a need for more validation.

https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4768 for the two fixes I could see as necessary as of now

**Pull request merged**